playbooks/archivematica-centos7/singlenode.yml

```diff
@@ -7,6 +7,16 @@
       tags:
         - "always"
 
+    - name: "Install some python packages"
+      become: "yes"
+      yum:
+         name: "{{ item }}"
+         state: "present"
+      with_items:
+        - libsemanage-python
+        - policycoreutils-python
+        - MySQL-python
```

Comment:
I don't think this playbook is a good place to add extra os level dependencies.  Could these be added instead here: https://github.com/artefactual/archivematica/tree/stable/1.7.x/src/archivematicaCommon/osdeps ?  Are these dependencies only required when deploying Archivematica using ansible, or are they required for an Archivematica instance that was manually installed (from rpm packages for example)?